pages/common/ls.md

```diff
@@ -25,3 +25,7 @@
 - Long format list of all files, sorted by modification date (oldest first):
 
 `ls -ltr`
+
+- Quote strange filenames (delete using `rm -- NAME`):
```

Comment:
Can we define "strange" ? Perhaps non-printable characters is better. Also, is adding the delete statement necessary ?

Reply:
I think that `non-printable` would be better, if this is what they are.